gen/aws/templates/cloudformation.json

```diff
@@ -71,10 +71,10 @@
         "stable": "ami-05bc0164"
       },
       "us-west-1": {
-        "stable": "ami-27553a47"
```

Comment:
@spahl @cmaloney has valid points here. We probably should looking at updating the base AMI ASAP else this might become a blocker?